app/src/main/java/io/digitallibrary/reader/catalog/catalog_db.kt

```diff
@@ -151,15 +244,62 @@ interface BookDownloadDao {
     @Query("DELETE FROM book_downloads WHERE book_id = :bookId")
     fun delete(bookId: String)
 
-    @Query("DELETE FROM book_downloads WHERE book_id = :bookId")
+    @Query("DELETE FROM book_downloads WHERE book_id in (:bookId)")
```

Comment:
What's the difference?

Reply:
":bookId" is translated to list of bookIds with commas between them, since the parameter is a list. I should rename the parameter to bookIds. (The sql in the old version is invalid and crashed the app.)